core/market/decentralized/Cargo.toml

```diff
@@ -21,6 +21,7 @@ ya-service-api-interfaces = "0.1"
 ya-service-api-web = "0.1"
 ya-service-bus = "0.2"
 
+actix = "0.9"
```

Comment:
we do not use actix at all within decentralized Market and now we can also go without it